.gitignore

```diff
@@ -3,4 +3,5 @@ Cargo.lock
 .idea
 snapshot_before.png
 snapshot_after.png
-documents_example
\ No newline at end of file
+documents_example
+bacon.toml
```

Comment:
wot?

Reply:
https://dystroy.org/bacon/ ?

Reply:
yep its that, rust-analyzer doesnt seem to work well